cmake/public/threads.cmake

```diff
@@ -5,14 +5,12 @@ if(THREADS_FOUND AND NOT TARGET Threads::Threads)
   add_library(Threads::Threads INTERFACE IMPORTED)
 
   if(THREADS_HAVE_PTHREAD_ARG)
-    set_property(
-        TARGET Threads::Threads
-        PROPERTY INTERFACE_COMPILE_OPTIONS "-pthread")
+    set_property(TARGET Threads::Threads
+                 PROPERTY INTERFACE_COMPILE_OPTIONS "-pthread")
```

Comment:
Errr, this is a no-op?

Reply:
Yes. My cmake formatter has formated this. Is that a problem?

Reply:
OK, just making sure you didn't want an actual substantive change here.